docs/getting-started/istio/index.md

```diff
@@ -55,15 +68,27 @@ spec:
   hosts:
   - rollouts-demo.local
   http:
-  - name: primary  # Should match spec.strategy.canary.trafficRouting.istio.virtualService.routes
+  - name: http-primary  # Should match spec.strategy.canary.trafficRouting.istio.virtualService.routes
```

Comment:
spec.strategy.canary.trafficRouting.istio.virtualService.routes -> `rollout.spec.strategy.canary.trafficRouting.istio.virtualService.routes`

Reply:
Will change! Thanks.

Reply:
@huikang I've addressed all the README related feedback in #1414